Sources/BraintreeDataCollector/BTDataCollector.swift

```diff
@@ -0,0 +1,124 @@
+import Foundation
+import CoreLocation
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+#if canImport(BraintreeKountDataCollector)
+import BraintreeKountDataCollector
+#endif
+
+/// Braintree's advanced fraud protection solution.
+@objcMembers public class BTDataCollector: NSObject {
+    
+    /// The Kount SDK device collector, exposed internally for testing
+    var kount: KDataCollector?
+
+    private var fraudMerchantID: String?
+    private let apiClient: BTAPIClient
+    
+    // TODO: overriding load() has been deprecated for a while. Right now we are using it to load PPDataCollector if needed in this class. Since these 2 classes will be combined, leaving this out for now.
+    
+    ///  Initializes a `BTDataCollector` instance with a `BTAPIClient`.
+    /// - Parameter apiClient: An instance of `BTAPIClient`
+    @objc(initWithAPIClient:)
+    public init(apiClient: BTAPIClient) {
+        self.apiClient = apiClient
+        super.init()
+        setUpKountWithDebugOn(false)
+    }
+    
+    // MARK: Public methods
+    
+    /// Set your fraud merchant id.
+    /// - Parameter merchantID: The fraudMerchantID you have established with your Braintree account manager.
+    /// - Note: If you do not call this method, a generic Braintree value will be used.
+    public func setFraudMerchantID(_ merchantID: String) {
+        fraudMerchantID = merchantID
+        kount?.merchantID = Int(merchantID) ?? 60000
```

Comment:
qq: Does the `60000` value have any significance? We could extract this into a `DEFAULT_KOUNT_MERCHANT_ID` constant.

Reply:
Yeah, that's the default value provided by the gateway if no merchant ID is passed to us. It used to be the shared kount merchant ID for merchants using Kount Standard before that was replaced. I can extract this out into a constant though for sure, that's a good call.